### PR TITLE
Added new GMPE

### PR DESCRIPTION
**Pull request by mleonardos**
Implementation of the GMPE derived by Arroyo et al., (2010) for Mexican interface earthquakes.

### CODE REVIEW

**Comment by mleonardos** [comment]
#7403 Fixed bug in the validation tables of AR10. A test was run to confirm that the issue has been solved.
hazardlib_test

File: openquake/hazardlib/gsim/arroyo_2010.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    "Strong ground-motion relations for Mexican interplate earhquakes", 
+    J. Seismol., 14:769-785.
+
+    The original formulation predict peak ground acceleration (PGA), in
+    cm/s**2, and 5% damped pseudo-acceleration response spectra (PSA) in 
+    cm/s**2 for the geometric average of the maximum component of the two 
+    horizontal component of ground motion.
+
+    The GMPE predicted values for Mexican interplate events at rock sites
+    (NEHRP B site condition) in the forearc region.
```
Comment (mmpagani) [inline openquake/hazardlib/gsim/arroyo_2010.py:74]: Perhaps you could add a short note to explain how you created the verification tables.

**Review by mmpagani — APPROVED** [review]
Apart from the minor comment on the verification tables, LGTM. Thanks a lot for your contribution.

File: openquake/hazardlib/gsim/arroyo_2010.py (hunk 1 above)
Comment (mleonardos) [reply openquake/hazardlib/gsim/arroyo_2010.py:74]: Thank you for your comment. I added a short note as suggested in the code: arroyo_2010_test.py. Please, let me know if it is correct.

**Review by micheles — APPROVED** [review]

**Pull request merged**